### ISSUE TITLE
trouble with unpacking files with $ in names

### ISSUE THREAD

**Issue by volth** [elfmz/far2l#495]
Example archive:
http://central.maven.org/maven2/org/jetbrains/kotlin/kotlin-compiler/1.3.21/kotlin-compiler-1.3.21.jar

![](http://i.imgur.com/8mqVtTT.png)

If I copy ```ClasspathRootsResolver$addModularRoots$1.class``` to the left panel using ```F5``` copy, ```ClasspathRootsResolver.class``` is copied instead.

**Comment by volth** [comment]
```F3``` also opens wrong file

**Comment by denis-adamchuk** [comment]
Looks like it has nothing to do with far2l.
I have c.zip and c.7z files, each of them contains "cleanup.sh" and "cleanup.$sh" files.
When I try 
   /home/sandbox$ 7z e c.7z "cleanup.$sh"
7-Zip reports _No files to process_
When I try
   /home/sandbox$ unzip c.zip "cleanup.$sh"
unzip reports _caution: filename not matched:  cleanup._

**Comment by volth** [comment]
@denis-adamchuk try ```'cleanup.$sh'``` (single quoted) or ```"cleanup.\$sh"```

```"cleanup.$sh"``` in bash == ```"cleanup." . $ENV{sh}``` in perl

**Comment by denis-adamchuk** [comment]
Yes, it works. May be far2l should escape $ character when passing a file name to the archiver, what do you think?

**Comment by unxed** [comment]
> Example archive:
> http://central.maven.org/maven2/org/jetbrains/kotlin/kotlin-compiler/1.3.21/kotlin-compiler-1.3.21.jar
> 
> ![](https://camo.githubusercontent.com/115839056ce20ab3808d6a2307544e9bd733c805/687474703a2f2f692e696d6775722e636f6d2f386d71567454542e706e67)
> 
> If I copy `ClasspathRootsResolver$addModularRoots$1.class` to the left panel using `F5` copy, `ClasspathRootsResolver.class` is copied instead.

retested with https://repo1.maven.org/maven2/org/jetbrains/kotlin/kotlin-compiler-embeddable/1.3.21/kotlin-compiler-embeddable-1.3.21.jar

looks to be solved now
